NEWS.md

```
@@ -60,38 +65,44 @@ milestone for 3.8.0
+  Deprecate pgr_analyzeOneWay in 3.8
+* [#2826](https://github.com/pgRouting/pgrouting/issues/2826):
+  Deprecate pgr_createVerticesTable in 3.8
 
-* Utilities
+Deprecated functions:
 
-  * [#2848](https://github.com/pgRouting/pgrouting/issues/2848):
-    pgr_separateCrossing
-  * [#2849](https://github.com/pgRouting/pgrouting/issues/2849):
-    pgr_separateTouching
+- Migration section is created.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)